plugins/woocommerce/src/Internal/Features/FeaturesController.php

```diff
@@ -123,7 +123,7 @@ public function __construct() {
 			),
 		);
 
-		$this->legacy_feature_ids = array( 'analytics', 'new_navigation' );
+		$this->legacy_feature_ids = array( 'analytics', 'new_navigation', 'product_block_editor' );
```

Comment:
We should really add an extra option to the above feature config around dependencies. `plugin_support_required` or something like that.
But this will do for now. @octaedro could you create a separate issue to update this with something like the above suggested option?